index.html

```diff
@@ -18,92 +18,24 @@
 </div>
 
 <div class="row">
-  <div class="small-12 columns">
```

Comment:
I don't remember why I removed this specifically (likely faffing around with the pricing table width), though `small-12` is just `width: 100%`, so I _think_ it only makes sense if you're setting different widths at different breakpoints, right? Otherwise the `div`'s width is always 100%.